### PR TITLE
CLI `install`

### PR DESCRIPTION
**Pull request by alexeymorozov**
See #91

### CODE REVIEW

File: lib/fontist/font.rb [hunk 1]
```diff
@@ -50,6 +56,58 @@ def check_or_create_fontist_path!
...
+    end
+
+    def available_fonts
+      available_by_style || available_by_font || available_by_formula
+    end
+
+    def available_by_style
+      return unless @style
+
+      if formula
```
Comment (ronaldtse) [inline lib/fontist/font.rb:72]: How about shortcutting it with `return unless formula`?

File: lib/fontist/font.rb [hunk 2]
```diff
@@ -50,6 +56,58 @@ def check_or_create_fontist_path!
...
+            f.styles.select do |s|
+              s.font if s.type.casecmp?(@style)
+            end
+          end
+        end.flatten
+      end
+    end
+
+    def available_by_font
+      if formula
```
Comment (ronaldtse) [inline lib/fontist/font.rb:84]: How about shortcutting it with `return unless formula`?

File: lib/fontist/font.rb [hunk 3]
```diff
@@ -50,6 +56,58 @@ def check_or_create_fontist_path!
...
+            f.styles.map do |s|
+              s.font
+            end
+          end
+        end.flatten
+      end
+    end
+
+    def available_by_formula
+      if formula
```
Comment (ronaldtse) [inline lib/fontist/font.rb:96]: How about shortcutting it with `return unless formula`?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: lib/fontist/font.rb (hunk 3 above)
Comment (alexeymorozov) [reply lib/fontist/font.rb]: Highlighted your comment in a new PR

https://github.com/fontist/fontist/pull/149#discussion_r498748029

**Review by ronaldtse — APPROVED** [review]
Thanks @alexeymorozov !

**Pull request merged**